lib/shoulda/matchers/independent/delegate_method_matcher.rb

```diff
@@ -176,6 +173,9 @@ def delegate_method(delegating_method)
 
       # @private
       class DelegateMethodMatcher
+        autoload :StubbedTarget, 'shoulda/matchers/independent/delegate_method_matcher/stubbed_target'
+        autoload :DelegateObjectNotSpecified, 'shoulda/matchers/independent/delegate_method_matcher/target_not_defined_error'
```

Comment:
Never noticed that this constant isn't named the same as the file, whoops! 😅

Reply:
Should I rename the class or the file?

Reply:
I think this is fine as-is — some future update can fix this.